src/abstracts/contract-verifier.huff

```diff
@@ -83,69 +121,53 @@
         CHECK_GENERIC_CALL()
 }
 
-
-
-#define macro CHECK_VERIFIER() = takes(1) returns (1) {
-    dup1 verifier_ok jumpi
-
-    // console.log('no call verifier set')
-    0x6e6f2063616c6c20766572696669657220736574000000000000000000000000
-    0x0000000000000000000000000000000000000000000000000000000000000014
-    CONSOLE_LOG()
-
-    0x00 0x00 revert
-
-    verifier_ok:
-}
-
-/**
-Checks that a call can be performed
- -> taking stack [callOpcode, callContract, sentValue, argOffset, argSize]
-*/
+/// @dev Checks that a call can be performed
+///      -> taking stack [callOpcode, callContract, sentValue, argOffset, argSize]
+///
+/// Arguments :
+///  - function selector (0x04 length... signature of "verifyCall(uint1,address,uint256,bytes,bool)")
+///  - opcode (0x20 length packed)
+///  - contract address (0x20 length packed)
+///  - value sent (0x20 length packed)
+///  - copy of the first 10 words of calldata:
+///      * pointer to data (len 0x20)
+///      * array len (len 0x20)
+///      * actual data (len 0x140)
+///  - isTruncated (0x20 length packed)
+///
+/// Note : Do not use CONSOLE_LOG() in this macro because it writes to 
+///        CALLARGS_MSLOT (avoids allocating debug memory).
 #define macro CHECK_GENERIC_CALL() = takes(4) returns (0) {
-
-
-    //  ==========> /!\ WARNING: do not use CONSOLE_LOG() in this macro  /!\ <=========
-    //  because it writes to CALLARGS_MSLOT (avoids allocating debug memory)
-
-    //  =>  read the doc of CALLARGS_MSLOT constant to understand the memory layout written here.
-
     // copy opcode to args
     [CALLARGS_MSLOT_opcode] mstore // => [callContract, sentValue, argOffset, argSize]
+
     // copy contract to args
     [CALLARGS_MSLOT_contract] mstore // => [sentValue, argOffset, argSize]
+
     // copy sent value to args
     [CALLARGS_MSLOT_value] mstore // => [argOffset, argSize]
 
-
     // store "isTrucated" as false
     0x0 [CALLARGS_MSLOT_isTruncated] mstore
 
-
-    // -> here, we have [argOffset, argSize] on stack
+    // stack => [argOffset, argSize]
```

Comment:
I would put the stack `[argOffset, argSize]` at the end of the line above 
From what I saw and like, putting the state of the stack at the end of a line is nice
Example in huff mate [here](https://github.com/pentagonxyz/huffmate/blob/main/src/tokens/ERC20.huff#L85) or doc [here](https://docs.huff.sh/get-started/huff-by-example/#custom-errors)  
All stack state should be with the same space from code if possible